### DISCUSSION TITLE
Re-new Github Student Developer Pack

### DISCUSSION THREAD

**Discussion by sibtainali** [community/community#82775] [GitHub Education]
I want to renew my GitHub Student Developer Pack, but my university doesn't give us individual email addresses. We have an address like fa21-bcs-021@must.edu.pk on the university domain so we can login on CMS but we can't receive any email on it. Only teachers can receive it. Can I use something else, like my student card, to prove I'm a student? This might be a problem for other students too.

Regards
Subtain

**Comment by dizzpy** [comment]
1. Choose the "I don't have a school-issued email" option during verification.
2. Upload a clear photo or scan of any of these: Student ID card, enrollment verification letter, class schedule, or transcript.
3. Wait for GitHub to review and verify your student status, usually taking a few days.

**Comment by samus-aran** [comment]
Hi @sibtainali 

If you believe your applied school's details are incorrect please raise a ticket with [Education support ](https://support.github.com/contact/education)using the dropdown "My selected school has incorrect or incomplete information e.g. domains or campus location". 👍 All applications must go through [GitHub Global Campus Student Application form](https://education.github.com/discount_requests/student_application). We are unable to provide manual reviews or upload links for individual applicants but we can review if the school details are accurate.

You should provide evidence to support if your school has additional academic domains, has changed its academic email issuing policy, or does not provide academic emails. Reach out to your institution's IT team for this. 

Hope this helps! Let us know how you get along 👍
